src/core/agent/services/multiSigService.ts

```diff
@@ -1093,6 +1096,219 @@ class MultiSigService extends AgentService {
 
     return { op, exnSaid: exn.ked.d };
   }
+
+  async offerPresentMultisigACDC(
```

Comment:
`offerAcdcFromApply` to match the single sig naming

Reply:
Actually, that function is in IPEX communication service. Can we move this there too? I'd prefer to keep IPEX in one place.

So we can have `offerAcdcFromApply` and `multisigOfferAcdcFromApply`